### DISCUSSION TITLE
fr/integrations/edge-tpu/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#16985] [Docs]
# fr/integrations/edge-tpu/

Learn how to export YOLO11 models to TFLite Edge TPU format for high-speed, low-power inferencing on mobile and embedded devices.

https://docs.ultralytics.com/fr/integrations/edge-tpu/

**Comment by manu13008** [comment]
Hi,
Once I exported my best.pt model into edgetpu tflite, I tried to run an inference on an image as the code above but I got the following error : 

OSError: libedgetpu.so.1: cannot open shared object file: No such file or directory

Do you have any insights of how resolving this ? 

Thanks :)

**Reply by glenn-jocher** [reply]
@manu13008 it seems like the error is due to the missing Edge TPU runtime library. Please ensure you have installed the Edge TPU runtime on your device. You can find installation instructions in the [Edge TPU documentation](https://coral.ai/docs/accelerator/get-started/). If you need further assistance, feel free to check the resources there.
